src/vorta/views/archive_tab.py

```diff
@@ -365,14 +367,18 @@ def on_selection_change(self, selected=None, deselected=None):
             reason = reason or self.tr("(Select exactly one archive)")
 
             # too few or too many selected.
-            self.fArchiveActions.setEnabled(False)
+            self.bMountArchive.setEnabled(False)
+            self.bExtract.setEnabled(False)
+            self.bRename.setEnabled(False)
 
             for index in range(layout.count()):
                 widget = layout.itemAt(index).widget()
                 tooltip = widget.toolTip()
 
                 tooltip = self.tooltip_dict.setdefault(widget, tooltip)
-                widget.setToolTip(tooltip + " " + reason)
+
+                if 'Refresh' not in tooltip:
```

Comment:
Is this a dirty hack? I thought this was the only way to do this with the layout. Otherwise I could do something like https://github.com/borgbase/vorta/pull/1723/files#diff-1340facde0499b13accf612e9eab00d2c0909090fb7320e61ce2f0cf6282ca02R370-R372

Reply:
This link goes to the current page, but you're right. This can be solved better probably.

Reply:
`if 'Refresh' not in tooltip:`

This one doesn't look very stable if we ever rename the tooltip text. Maybe there is a better way to use some metadata or name of the button instead.

Reply:
Thanks, Done.